static/js/util/validation.js

```diff
@@ -158,6 +154,10 @@ export function employmentValidation(profile: Profile): ValidationErrors {
         moment(workHistory.end_date).isBefore(workHistory.start_date, 'month')) {
         _.set(errors, ['work_history', String(index), 'end_date'], "End date cannot be before start date");
       }
+      if (!isNilOrEmptyString(workHistory.end_date) && workHistory.end_date !== undefined &&
```

Comment:
Yeah, I think with an older version of flow we had to add that to get it to pass. removed it.